src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -204,10 +206,11 @@ def setUp(self):
 
         # Create host files
         self.hostfile_servers = write_host_file.write_host_file(
-            self.hostlist_servers, self.workdir)
+            self.hostlist_servers, self.workdir, self.hostfile_servers_slots)
```

Comment:
Does this create the correct hostfile for mpich?

If this test uses the hydra job manager (mpirun->mpiexec.hydra) the format for the host file is host:ppn,  not host slots=ppn.   Orterun has the format host slots=ppn.